framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -529,8 +529,7 @@ protected void addInformationFromPreconditions(
                                 false);
                 info.insertValue(expr, annotation);
             } catch (FlowExpressionParseException e) {
-                // report errors here
-                analysis.checker.report(e.getResult(), methodTree);
```

Comment:
Similar code is further down in `processPostconditions` and `processConditionalPostconditions`.
Shouldn't those also be changed?

More generally, I think no error reporting should be done as part of flow analysis. Not all type systems use flow and checks should be implemented in the visitor. Flow refinement strictly improves the precision, but doesn't report additional bugs.

In `org.checkerframework.framework.flow.CFAbstractAnalysis#checker` the documentation states: `A checker used to do error reporting.`
Once we remove the three error reports with this PR, the only other uses of that field are to look up options and warning suppressions.
Can we update the documentation of that field as part of this PR, to document that we don't want any use of `checker.report` in the flow part.